### PR TITLE
Doc for win and no vcpkg

### PR DESCRIPTION
**Pull request by vhvb1989**
fixes: https://github.com/Azure/azure-sdk-for-c/issues/429

### CODE REVIEW

File: az-keyvault-config.cmake.in [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# - Config file for the az-keyvault package
```
Comment (ahsonkhan) [inline az-keyvault-config.cmake.in:1]: What's the purpose of these *.in files?

File: CMakeLists.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ cmake_minimum_required (VERSION 3.10)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake-modules")
 
+include(CMakePackageConfigHelpers)
```
Comment (ahsonkhan) [inline CMakeLists.txt:8]: Why do we need this?

File: README.md [hunk 3]
```diff
@@ -301,6 +301,8 @@ cd vcpkg
 
 If you previously installed VCPKG and dependencies, you may need to run `.\vcpkg.exe upgrade --no-dry-run` to upgrade to the latest packages.
 
+> Note: Setting up a development environment in windows without VCPKG is not recommended. It requires installing all dev-dependencies globally and manually setting cmake files to link each of them.
```
Comment (ahsonkhan) [inline README.md:304]:
```suggestion
> Note: Setting up a development environment in windows without VCPKG is not supported. It requires installing all dev-dependencies globally and manually setting cmake files to link each of them.

```

File: CMakeLists.txt (hunk 2 above)
Comment (vhvb1989) [reply CMakeLists.txt:8]: Not sure why it is added  XD...
Need to revert this.. sorry

Comment (vhvb1989) [reply CMakeLists.txt]: my master branch on my fork got corrupt with unwanted changes. Sorry about it

File: README.md [hunk 4]
```diff
@@ -301,6 +301,9 @@ cd vcpkg
 
 If you previously installed VCPKG and dependencies, you may need to run `.\vcpkg.exe upgrade --no-dry-run` to upgrade to the latest packages.
 
+> Note: Setting up a development environment in windows without VCPKG is not supported. It requires installing all dev-dependencies globally and manually setting cmake files to link each of them.
+
+
```
Comment (ahsonkhan) [inline README.md:306]: nit: Remove extra new line. If you use a markdown linter extension in VSCode, it would catch such changes.

**Pull request merged**